meltano/edk/process.py

```diff
@@ -68,21 +68,27 @@ def run(
         stdout and stderr by default are set up to use subprocess.PIPE. If you do not
         want to capture io from the subprocess use subprocess.DEVNULL to discard it.
 
+        The Invoker's at env and cwd are used when calling subprocess.run. If you want
+        to override these you're likely better served using subprocess.run directly.
+
+        Lastly note that this method is blocking AND subprocess.run is called with
+        check=True. This means that if the subprocess fails a CalledProcessError will
+        be raised.
```

Comment:
Our docstrings should be treated as markdown, as that's how we'll render them when we set up code documentation generation via Sphinx or Mkdocstrings.

I'd add the suggestions for the other parts of the docstring, but GitHub will only let me add suggestions for the diff.

```suggestion
        The Invoker's at env and cwd are used when calling `subprocess.run`. If you want
        to override these you're likely better served using `subprocess.run` directly.

        Lastly note that this method is blocking AND `subprocess.run` is called with
        `check=True`. This means that if the subprocess fails a `CalledProcessError` will
        be raised.
```